mentor.html

```diff
@@ -3,40 +3,150 @@
+          this strange new world. It's your duty that the plan the mentee comes
+          up with, is do-able, provides orientation and is technically sound.
+          You're the expert!
+        </p>
+        <h3>5. Track Progress</h3>
+        <p>
+          We encourage you to set up regular milestones in your plan that get
+          reviewed and celebrated by you and your mentee. It's important to
+          understand where you are and how you progress. Additionally we believe
+          that meeting as often as you can will help, over time, to build the
+          relationship and bonding with your mentee. The consistency of your
+          relationship is very important. Find a regular schedule where you meet
+          or even better meet as often as you can!
+        </p>
+        <h3>6. Use the community</h3>
+        <p>
+          Our community consists of very experienced people and is
+          one of the biggest and diverse free software projects on this planet.
+          We provide well thought out structures, processes and infrastructure
+          that reflect our experience with introducing new contributors since
+          day one of our existence. openSUSE believes that one of the most
+          important tasks is to bring in new hackers with new perspectives,
+          backgrounds and ideas into our community. Make sure that you make use
+          of this environment to the full extent!
+        </p>
+        <h3>7. Be Agile</h3>
+        <p>
+          The only thing that is constant is change! You need to adapt your plan,
+          milestones, time line, the technical details, your behavior and
+          attitude during your time. Respond to change, help your mentee to do
+          the same.
+        </p>
+        <h3>8. Collaborate</h3>
+        <p>
+          Whatever challenge you face, remember you are not alone! You mingle on
+          the shoulders of giants together with many other openSUSE mentors.
+          Make sure you share your experience, your ideas and your problems with
+          them. All of you strive to reach the same goal: getting people into
+          free software. Make sure you work together in an open, transparent and
+          friendly manner as part of our worldwide community.
+        </p>
+        <h3>9. Want this</h3>
+        <p>
+          Mentoring requires time, effort and endurance. It will cost you some
+          evening you wanted to go out with your friends, at least an hour every
+          day and a lot of nerves because you explain things that are obvious to
+          you over and over again. If you're not sure that you are willing to do
+          this then it's probably better you don't do it. Once you become a mentor
+          you make a promise to your future mentee that you do the best you can.
+          Half ass mentor jobs scare off an individual from the wonderful world
+          of free software!
```

Comment:
It would be great to also add some of the positive parts of being a mentor, the satisfaction of being able to help others to grow into the community, achieving results for the project, the opportunity to learn together, and sometimes making friends for life.